### PR TITLE
Create Privledges and added Acads table by v1krant

### PR DESCRIPTION
**Pull request by vikrantmalik051**
Code for populating table added

Code for Populating users table

Updating gitignore

Update .gitignore

Created Tags model

Privledges and acads table added

Acads and Privileges added

Create Users Model Table

Code for populating table added

Code for Populating users table

Updating gitignore

Update .gitignore

Created Tags model

Update settings.py

Acads, privileges added and commits squashed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.DS_Store
 __pycache__/
-/lifeiitkbackend/__pycache__
+*.pyc
```
Comment (ditttu) [inline .gitignore]: new line here

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
+.DS_Store
```
Comment (ditttu) [inline .gitignore]: Also, remove .DS_Store

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 __pycache__/
-/lifeiitkbackend/__pycache__
```
Comment (ditttu) [inline .gitignore]: Why was this removed?

**Review by ditttu — CHANGES_REQUESTED** [review]

**Comment by goyal-apoorv** [comment]
Acads table needs to have a field for weekdays on which classes will be held
